lib/trento/domain/sap_system/application.ex

```diff
@@ -11,6 +13,7 @@ defmodule Trento.Domain.SapSystem.Application do
 
   deftype do
     field :sid, :string
+    field :ensa_version, Ecto.Enum, values: EnsaVersion.values(), default: EnsaVersion.no_ensa()
```

Comment:
Setting the value in the `Application` field, as this term is related to the Application part.
It could very well go in the SAP system aggregate as well, as a SAP system only has one application, but it looked more appropriate to put it here to have better context